.github/workflows/nix.yml

```diff
@@ -1,6 +1,24 @@
 name: nix
 on: [push, pull_request]
 
+# This workflow has two job matrixes.
+# They can be considered phases because the second matrix ("test")
+# depends on the first ("dependencies").
+#
+# The first phase has a job in the matrix for each combination of
+# variables that affects dependency ABI:
+# platform, compiler, and configuration.
+# It creates a GitHub artifact holding the Conan profile,
+# and builds and caches binaries for all the dependencies.
+# If an Artifactory remote is configured, they are cached there.
+# If not, they are added to the GitHub artifact.
+# GitHub's "cache" action has a size limit that is too small.
```

Comment:
My understanding is that the cache limit is 10Gb total. These artifacts are < 200Kb. Is that only because the Artifactory remote is available? If so, I suggest rephrasing this as `GitHub's "cache" action has a size limit that is too small to hold the binaries if they are built locally.`